common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -186,68 +186,126 @@ def get_problem(self):
         """
         xml = dedent("""
             <problem>
-            <p>question text</p>
-            <stringresponse answer="A">
-                <stringequalhint answer="C"><a href="#">aa bb</a> cc</stringequalhint>
-                <textline size="20"/>
-            </stringresponse>
-            <demandhint>
-              <hint>aa <a href="#">bb</a> cc</hint>
-              <hint><a href="#">dd  ee</a> ff</hint>
-            </demandhint>
+                <question>
+                    <p>question text</p>
+                    <stringresponse answer="A">
+                        <stringequalhint answer="C"><a href="#">aa bb</a> cc</stringequalhint>
+                        <textline size="20"/>
+                    </stringresponse>
+                    <demandhint>
+                      <hint>question 1 hint 1</hint>
+                      <hint>question 1 hint 2</hint>
+                    </demandhint>
+                </question>
+                <question>
+                    <p>That is the question</p>
+                    <multiplechoiceresponse>
+                      <choicegroup type="MultipleChoice">
+                        <choice correct="false">Alpha <choicehint>A hint</choicehint></choice>
+                        <choice correct="true">Beta</choice>
+                      </choicegroup>
+                    </multiplechoiceresponse>
+                    <demandhint>
+                      <hint>question 2 hint 1</hint>
+                      <hint>question 2 hint 2</hint>
+                    </demandhint>
+                </question>
             </problem>
         """)
         return XBlockFixtureDesc('problem', 'PROBLEM HTML HINT TEST', data=xml)
 
     def test_check_hint(self):
         """
-        Test clicking Check shows the extended hint in the problem message.
+        Scenario: Test clicking Check shows the extended hint in the problem message.
+        Given I am enrolled in a course.
+        And I visit a unit page with two CAPA question
+        Then I gave incorrect answers for both questions
+        When I click the check button
+        Then I should see 2 hint messages
```

Comment:
Given that the framework exists for having bok choy tests with 2 problems on the same page, add an a11y test with that configuration (which should fail presently due to the non-unique IDs).